tests/integration/components/sticky-element-test.js

```diff
@@ -346,6 +346,41 @@ module('Integration | Component | sticky element', function(hooks) {
       assert.dom('#debug').hasText(debug, debug);
     });
 
+    test(`Is resizable | Size: ${testCase.size}, offView: ${testCase.offView}, stick to bottom: ${testCase.stickToBottom === false ? 'false' : 'true'}, scroll position: ${testCase.scrollPosition}`, async function(assert) {
+      let stickyElementWidth;
+      let containerWidth;
+      this.setProperties(testCase);
+      this.set('bottom', testCase.stickToBottom ? 0 : null);
+      this.set('largeContainer', true);
+      await render(hbs`
+        <div class="row">
+          <div class="col {{size}} {{if offView "off"}} {{if largeContainer "large-container" "small-container"}}">
+            {{#sticky-element class="sticky" bottom=bottom as |sticky|}}
+              <p id="debug">
+                {{sticky-debug sticky}}
+              </p>
+            {{/sticky-element}}
+          </div>
+        </div>
+      `);
+
+      let debug = output(testCase.sticky);
+
+      await scrollTo(testCase.scrollPosition);
+      stickyElementWidth = document.querySelector('.sticky-element').clientWidth;
+      containerWidth = document.querySelector('.col').clientWidth;
+      assert.equal(stickyElementWidth, containerWidth, 'Sticky element 500px width');
```

Comment:
In case the change of layout (`this.set('largeContainer', false);`) would not work, this would still pass I guess. Maybe better to use hard-coded values (`300px` / `500px`) in tests, so we can be sure it really changes the size?